src/main.js

```diff
@@ -27,10 +27,10 @@ exports.main = async function main() {
 
     const version = getVersionById(versions, targetVersion)
 
-    if (version == null) {
-      core.error('No log entry found.')
-      core.setOutput('log_entry', '')
-      return
+    if (version == null && targetVersion != null) {
```

Comment:
Only refactor here but I would prefer either:

```js
if (version == null) {
  if (targetVersion == null) {
    throw new Error(`No log entry found`)
  }

  throw new Error(`No log entry found for version ${targetVersion}`)
}
```
or 
```js
if (version == null) {
  throw new Error(`No log entry found ${
    targetVersion != null
      ? `for version ${targetVersion}`
      : ``
  }`)
}
```

Reply:
@mindsers I updated this to the second option you proposed here.